### PR TITLE
feat(health): Add "drilldowns" for Browsers and Devices

### PR DESCRIPTION
**Pull request by billyvg**

### CODE REVIEW

File: src/sentry/static/sentry/app/views/organizationHealth/details.jsx [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+import {t} from 'app/locale';
+import LineChart from 'app/components/charts/lineChart';
+import PieChart from 'app/components/charts/pieChart';
+
+import HealthPanelChart from './styles/healthPanelChart';
+import HealthRequest from './util/healthRequest';
+import withHealth from './util/withHealth';
+
+class OrganizationHealthDetails extends React.Component {
```
Comment (billyvg) [inline src/sentry/static/sentry/app/views/organizationHealth/details.jsx]: TBD If we can keep this as a generic component, there may be specific components for each "detail" view.

File: src/sentry/static/sentry/app/views/organizationHealth/devices.jsx [hunk 2]
```diff
@@ -1,62 +1,89 @@
...
+                showLegend={false}
+                series={[
+                  {
+                    seriesName: title,
+                    data: tagData.map(([name, value]) => ({name, value})),
+                  },
+                ]}
+                title={title}
+              >
+                {({series}) => <PieChart height={300} selectOnRender series={series} />}
```
Comment (lynnagara) [inline src/sentry/static/sentry/app/views/organizationHealth/devices.jsx]: The selectOnRender behaviour is quite confusing to me (i.e. how after you move your mouse over it, it disappears permanently)

File: src/sentry/static/sentry/app/views/organizationHealth/eventsTableChart.jsx [hunk 3]
```diff
@@ -87,13 +97,29 @@ class EventsTableChart extends React.Component {
...
     );
   }
 }
 
+const Row = styled(function RowComponent({className, data, rowIndex, onClick, children}) {
+  return (
+    <div
+      className={className}
+      onClick={e =>
+        typeof onClick === 'function' && rowIndex > -1 && onClick(data[rowIndex], e)}
```
Comment (lynnagara) [inline src/sentry/static/sentry/app/views/organizationHealth/eventsTableChart.jsx]: How could a rowIndex ever be -1? I assume that this component would never be rendered in that case

**Review by lynnagara — APPROVED** [review]

File: src/sentry/static/sentry/app/views/organizationHealth/devices.jsx (hunk 2 above)
Comment (billyvg) [reply src/sentry/static/sentry/app/views/organizationHealth/devices.jsx]: Yeah it's something we should address in the future, unfortunately ECharts doesn't provide any great behavior by default and we'll have to develop something ourselves.

**Pull request merged**